shopfloor_reception_purchase_partner_ref_mobile/static/src/scenario/reception.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright 2023 Michael Tietz (MT Software) <mtietz@mt-software.de>
+ * License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+ */
+
+import {process_registry} from "/shopfloor_mobile_base/static/wms/src/services/process_registry.js";
+
+const reception_scenario = process_registry.get("reception");
```

Comment:
Can you please try a std Vue.js override? 
```js
Vue.component("reception", {
    "methods": ...
})
``` 

Then you should be able to use `this.$super(reception_scenario).manual_select_options_for_select_document()` to get the original result.

Reply:
Or did i miss something?

Reply:
I will resolve this, because @simahawk answered it already in an other PR. 
https://github.com/OCA/wms/pull/647#discussion_r1274454764